File: video/rest/rooms/create-peer-to-peer-room/create-peer-to-peer-room.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Video.V1;

class Example
{
    static void Main (string[] args)
    {
        // Find your Account SID and Auth Token at twilio.com/console
        const string apiKeySid = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string apiKeySecret = "your_api_key_secret";

        TwilioClient.Init(apiKeySid, apiKeySecret);

        var room = RoomResource.Create(
            uniqueName: "DailyStandup",
            type: RoomResource.RoomTypeEnum.PeerToPeer,
            enableTurn: true,
            statusCallback: new Uri("http://example.org"));

        Console.WriteLine(room.Sid);
    }
}

File: video/rest/rooms/create-peer-to-peer-room/create-peer-to-peer-room.5.x.rb
# Download the Ruby helper library from twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
api_key_sid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
api_key_secret = 'your_api_key_secret'

client = Twilio::REST::Client.new api_key_sid, api_key_secret

p2p_room = client.video.rooms.create(
  unique_name: 'DailyStandup',
  type: 'peer-to-peer',
  enable_turn: false,
  status_callback: 'http://example.org'
)

puts p2p_room.sid
